### ISSUE TITLE
Error on running "oclif readme"  ---> (cmd.args || []).filter is not a function #1082 "

### ISSUE THREAD

**Issue by sbansla** [oclif/oclif#1082]
Do you want to request a *feature* or report a *bug*?
-----------------------------------------------------
bug

What is the current behavior?
-----------------------------
If the current behavior is a bug, please provide the steps to reproduce.
We are using oclif in our cli app, 
1. Upgraded to oclif **3.5.0** version
2. Run from our repo: oclif readme
```
Error:
 oclif:readme commands: 27 +652ms
  oclif:readme rendering command autocomplete +5ms
Error: (cmd.args || []).filter is not a function
    at Object.error (/Users/sburman/2023/forks/twilio-cli/node_modules/oclif/node_modules/@oclif/core/lib/errors/index.js:27:15)
    at Readme.error (/Users/sburman/2023/forks/twilio-cli/node_modules/oclif/node_modules/@oclif/core/lib/command.js:130:23)
    at Readme.renderCommand (/Users/sburman/2023/forks/twilio-cli/node_modules/oclif/lib/commands/readme.js:136:18)
    at /Users/sburman/2023/forks/twilio-cli/node_modules/oclif/lib/commands/readme.js:115:39
    at Array.map (<anonymous>)
    at Readme.commands (/Users/sburman/2023/forks/twilio-cli/node_modules/oclif/lib/commands/readme.js:115:25)
    at Readme.run (/Users/sburman/2023/forks/twilio-cli/node_modules/oclif/lib/commands/readme.js:38:123)
    at async Readme._run (/Users/sburman/2023/forks/twilio-cli/node_modules/oclif/node_modules/@oclif/core/lib/command.js:107:22)
    at async Config.runCommand (/Users/sburman/2023/forks/twilio-cli/node_modules/oclif/node_modules/@oclif/core/lib/config/config.js:270:25)
    at async Object.run (/Users/sburman/2023/forks/twilio-cli/node_modules/oclif/node_modules/@oclif/core/lib/main.js:75:5)
```

What is the expected behavior?
------------------------------
There should not be any failure.
Please mention your oclif and OS version.
oclif **3.5.0** version

**Comment by mdonnalley** [comment]
@sbansla Thanks for reporting the issue. I'm taking a look at it now and hope to have a solution later this morning. In the meantime, you should be able to pin to version 3.4.6

**Comment by mdonnalley** [comment]
@sbansla We just published a patch to `@oclif/core` (1.24.3) that should resolve this issue for you. So you'll just need to update that dependency and then you'll be able to use `@oclif/3.5.0` again.

**Comment by sbansla** [comment]
@mdonnalley Thank you for quick fix.
